docs/introduction.md

```diff
@@ -0,0 +1,50 @@
+# Introduction
+
+AdvancedHMC.jl provides a robust, modular and efficient implementation of advanced HMC algorithms. 
+AdvancedHMC.jl provides a standalone implementation of NUTS that recovers and extends Stan’s NUTS algorithm. 
+AHMC is written in Julia, a modern high-level language for scientific computing, benefiting from optional 
+hardware acceleration and interoperability with a wealth of existing software written in both Julia and other 
+languages, such as Python.
+
+It is desirable to have a high quality, standalone NUTS implementation in a high-level language,
+for research on HMC algorithms, reproducible comparisons and real-world approximate 
+inference applications in different domains. To this end, we introduce
+AdvancedHMC.jl (AHMC), a robust, modular and efficient implementation of Stan’s NUTS
+and several other commonly used HMC variants in Julia.
+
+## Getting started 
```

Comment:
This is good. Other than this minimal example, we also have a few we want to include 

- multi-threading
- vectorized mode & its benchmarking
- gpu support
- diffeq intergrators

(please remind me if I miss any)

I think we need to decide if we want to have an extra page like "advanced usage" or something, and put (some of) them there.

Reply:
I plan to have a separate page for parallelism with examples since I believe that is a high priority for a lot of users. The diffeq integrators can be added to a "advanced usage" page, some more things that can be added there, more details about the specifics of implementation and a sort of developer documentation.